packages/@react-spectrum/tabs/src/Tabs.tsx

```diff
@@ -335,6 +339,8 @@ export function TabPanels<T>(props: SpectrumTabPanelsProps<T>) {
   const {tabListState} = tabState;
 
   const factory = useCallback(nodes => new ListCollection(nodes), []);
+  // TODO: how to fix this one?
```

Comment:
what was the issue here?

Reply:
added comment with the error i haven't been able to resolve yet

Reply:
Do we need to extend the TabPanel generic from object like we do for our other collection components? Then perhaps nodes needs to be defined as `(nodes: Iterable<Node<T>>)` in the factory callback?

Reply:
I pushed a fix